### ISSUE TITLE
Call template literal as normal function

### ISSUE THREAD

**Issue by luwol03** [chalk/chalk-template#2]
Hello,

I try to parse a string (e.g. `"hello {green world}"`) that contains the chalk template syntax.

My first try was to parse that in the template literal like this:
```js
const myString = "hello {green world}";
console.log(chalk`${myString}`);
```

but without success. It returned the exact string I passed in without the green color format. So I debugged a little bit and found out that following works.
```js
const d = ["hello {green world}"];
d.raw = [...d];
console.log(chalk(d));
```

But that looks very messy. Is there a better approche?

**Comment by aryan02420** [comment]
I currently use this. I am getting the template string from config file, so `` chalk`...` `` tag cannot be used.
Will this enhancement be a part of the main chalk repo too?

![image](https://user-images.githubusercontent.com/58023300/135723160-dad3c5f0-26f0-40bb-9834-bc2903ec62d2.png)

**Comment by aryan02420** [comment]
I dont really mind the extra two lines, however `chalk/source/templates.js` might change, without any change in the chalk API.
Something like `chalk.template("hello {red world}!")` would be nice

**Comment by luwol03** [comment]
The chalk cli also used that really messy raw thing I mentioned in the description above.

**Comment by Qix-** [comment]
The "really messy raw thing" is how you do what is being asked in Javascript. That's how tagged template literals work under the hood - it's the same way you'd call many other tagged template literal functions.

**Comment by Qix-** [comment]
`0.4.0` has just landed with this support - see #8. Thanks @GeorgeTaveras1231!
